src/main/java/org/folio/fqm/service/MigrationService.java

```diff
@@ -1,8 +1,13 @@
 package org.folio.fqm.service;
 
 import java.util.List;
+
+import lombok.RequiredArgsConstructor;
 import org.apache.commons.lang3.NotImplementedException;
+import org.springframework.stereotype.Service;
 
+@Service
+@RequiredArgsConstructor
 public class MigrationService {
 
   public int getLatestVersion() {
```

Comment:
Out of scope for this PR, probably, but I still think we should use something besides `int` for the version. I don't really want to have to remember to bump another version somewhere when we're already having to juggle the artifact version, API versions, and liquibase version

Reply:
something like this ```
@Value("${artifact.version}")
    private String artifactVersion;
```